modules/apim_devportal/templates/carbon-home/bin/wso2server.sh.erb

```diff
@@ -249,17 +249,19 @@ do
         CARBON_CLASSPATH="$CARBON_CLASSPATH":$f
     fi
 done
-for t in "$CARBON_HOME"/lib/commons-lang*.jar
+for t in "$CARBON_HOME"/lib/*.jar
 do
     CARBON_CLASSPATH="$CARBON_CLASSPATH":$t
 done
+
+
```

Comment:
Why these extra lines? Shall we fix them even if they are coming from the product?

Reply:
Fixed https://github.com/wso2/puppet-apim/pull/171/commits/2737c344e7a9f7ae3e376a0226d4788d5c1841d8